libbeat/kibana/client_config.go

```diff
@@ -36,8 +36,9 @@ type ClientConfig struct {
 	IgnoreVersion bool
 }
 
-var (
-	defaultClientConfig = ClientConfig{
+// DefaultClientConfig connects to a locally running kibana over HTTP
+func DefaultClientConfig() *ClientConfig {
```

Comment:
Let's return default configs by value. We do so in most places (or should) that construct a default configuration. Just by judging from the function interface returning by value tells you that the value all belongs to you and can be freely modified without worries.

Reply:
sounds good, i'll make that change now